server/middleware/setUpWebSecurity.ts

```diff
@@ -28,6 +28,7 @@ export default function setUpWebSecurity(): Router {
           styleSrc: ["'self'", (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`],
           fontSrc: ["'self'"],
           formAction: [`'self' ${config.apis.hmppsAuth.externalUrl}`],
+          upgradeInsecureRequests: process.env.NODE_ENV === 'development' ? null : [],
```

Comment:
Get it to work in Safari on localhost (I should suggest this for the template app...🤔)